packages/design-system/src/components/SimpleMenu/Menu.tsx

```diff
@@ -172,16 +174,41 @@ type ItemProps<TComponent extends React.ComponentType = typeof BaseLink> =
   | ItemInternalLinkProps<TComponent>
   | ItemExternalLinkProps;
 
-const MenuItem = ({ onSelect, disabled = false, isLink, ...props }: ItemProps) => {
+const MenuItem = ({ onSelect, disabled = false, isLink, startIcon, endIcon, isExternal, ...props }: ItemProps) => {
   return (
     <DropdownMenu.Item asChild onSelect={onSelect} disabled={disabled}>
-      {isLink || props.isExternal ? (
-        <OptionLink color="neutral800" {...props} isExternal={props.isExternal ?? false}>
```

Comment:
@HichamELBSI  Following the same thinking as your last comment I thought it would make sense to also destructure isExternal from props as well. Let me know though if I overlooked something.